prdoc/pr_5430.prdoc

```diff
@@ -0,0 +1,13 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: "pallet-collator-selection: use correctly the `WeightInfo` call for `new_session`"
```

Comment:
Maybe change the title of the PR as well.

```suggestion
title: "pallet-collator-selection: correctly register weight in `new_session`"
```

Reply:
Adapted 👍